lldp/rx.c

```diff
@@ -342,29 +342,68 @@ void rxProcessFrame(struct port *port, struct lldp_agent *agent)
 			}
 		}
 		if (tlv->type == TYPE_4) { /* port description */
-			agent->lldpdu |= RCVD_LLDP_TLV_TYPE4;
-			agent->rx.manifest->portdesc = tlv;
-			tlv_stored = true;
+			if (agent->lldpdu & RCVD_LLDP_TLV_TYPE4) {
+				LLDPAD_INFO("Received multiple port description"
```

Comment:
This is a user-visible change in the behavior.  I agree that we should drop the messages, as other implementations do, but there could be deployments with buggy transmitters that will start seeing failures.  We should document this change in the NEWS file.

Reply:
Sure. I will leave that to you since it looks like `ChangeLog` is normally only updated at release time.